### PR TITLE
docs: 06-14-2024-changelog

### PR DESCRIPTION
**Pull request by danieltprice**
Preview:
https://neon-next-git-docs-06-14-2024-changelog-neondatabase.vercel.app/docs/changelog/2024-06-14

### CODE REVIEW

File: content/changelog/2024-06-14.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+For example, this command compares the current schema state between the `main` branch and the development branch `dev/alex`:
+
+```bash
+neonctl branches schema-diff main dev/alex
+```
+
+Sample output of the `schema-diff` command might look like this:
+
+```sql
```
Comment (bgrenon) [inline content/changelog/2024-06-14.md:23]:
```suggestion
```diff
```

Comment (bgrenon) [reply content/changelog/2024-06-14.md:23]: I think the language code here needs to be "diff" for the inline `[!code ++] [!code --] `to take effect.

File: content/changelog/2024-06-14.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+If you installed this extension previously and want to upgrade to the latest version, please refer to [Update an extension version](/docs/extensions/pg-extensions#update-an-extension-version) for instructions.
+
+### Integration with Outerbase
+
+We are excited to announce that the [Outerbase](https://docs.outerbase.com) integration for Neon is now publicly available.
+
+This integration enables you to instantly connect your Neon Postgres database to Outerbase’s [Data Studio](https://www.outerbase.com/products/data-studio/) and invite your team members to view, edit, query, and visualize your data.
+
+Outerbase’s [AI integration](https://www.outerbase.com/products/ai/) takes this a step further by enabling users to perform complex SQL tasks using natural language, making data manipulation accessible even to those without advanced SQL knowledge.
```
Comment (bgrenon) [inline content/changelog/2024-06-14.md:69]:
```suggestion
Outerbase’s [AI integration](https://www.outerbase.com/products/ai/) takes this a step further by helping users perform complex SQL tasks using natural language, making data manipulation accessible even to those without advanced SQL knowledge.
```

File: content/changelog/2024-06-14.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+![Outerbase overview](/docs/relnotes/outerbase.gif)
+
+To learn how to connect Outerbase to your Neon project, check out [this guide](/docs/guides/outerbase).
+
+### Fixes & improvements
+
+- Fixed an issue with the dynamic rate limiter at the Neon Proxy that caused excessive CPU consumption and prevented new connections from being accepted.
+- Added a **Source** column to the **Branches** widget on the Project Dashboard showing icons that indicate the creation source for the branch. For example, you are now able to see if the branch was created in Neon, via the Neon Vercel Integration, or through our Hasura integration.
+- Added missing units of measure to the legends for several charts on the Neon **Monitoring** page in the Neon Console.
+- Added the timezone to the **Restore branch** modal. Previously, the modal only showed the date and time of the selected restore point.
```
Comment (bgrenon) [inline content/changelog/2024-06-14.md:82]: Add a bit more context?

Updated the **Restore branch** modal to include timezone information alongside the date and time of the selected restore point, providing clearer context for restore operations

**Review by bgrenon — APPROVED** [review]

File: content/changelog/2024-06-14.md (hunk 3 above)
Comment (danieltprice) [reply content/changelog/2024-06-14.md]: thank you!

**Pull request merged**